api/swagger.yaml

```diff
@@ -6277,6 +6277,7 @@ paths:
             - `name=<network-name>` Matches all or part of a network name.
             - `scope=["swarm"|"global"|"local"]` Filters networks by scope (`swarm`, `global`, or `local`).
             - `type=["custom"|"builtin"]` Filters networks by type. The `custom` keyword returns all user-defined networks.
+            - `dangling=["true"|"false"] Print only dangling networks`
```

Comment:
Yeah, I am just referenced to https://github.com/docker/docker/blob/master/api/swagger.yaml#L5089-L5091 for image prune dangling:

```
- `dangling=<boolean>` When set to `true` (or `1`), prune only
               unused *and* untagged images. When set to `false`
               (or `0`), all unused images are pruned.
```